projects/sandbox/train/pyproject.toml

```diff
@@ -9,8 +9,9 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 "bbhnet.data" = {path = "../../../libs/data", develop  = true}
-"bbhnet.trainer" = {path = "../../../libs/trainer", develop = true}
+"bbhnet.io" = {path = "../../../libs/io", develop = true}
```

Comment:
Can we remove the `io` lib dependency now that the tests are updated?

Reply:
Oh, sorry, I meant to move this back to a draft PR. I'm still figuring out how to check on the content of `validator`.

Reply:
I've got this updated in my local version.